src/main/kotlin/no/nav/familie/ba/mottak/task/JournalførSøknadTask.kt

```diff
@@ -20,6 +21,11 @@ class JournalførSøknadTask(private val pdfService: PdfService,
             val pdf = pdfService.lagPdf(task.payload)
             log.info("Generert pdf med størrelse ${pdf.size}")
             journalføringService.journalførSøknad(task.payload, pdf)
+        } catch (e: HttpClientErrorException.Conflict) {
```

Comment:
Er det mulig å lage tjenesten idempotent i stedet? Altså søke opp journalposten basert på externReferanseId før man journalfører og skipper hvis man alt har journalført? Slik at man aldri får denne feilen og tasken ikke blir rød hvis alt er i orden?

Reply:
Nei, finnes ikke endpoint for det som jeg kunne finne. Her er kommentaren der de sier at man skal sjekke på 409-status
https://github.com/navikt/dokarkiv/blob/e82662f70cfd55502d0eba65c554fa64d7d9925c/journalpost/src/main/java/no/nav/dokarkiv/journalpost/v1/api/opprettjournalpost/OpprettJournalpostRequest.java#L91

Reply:
Takk for review Stig.

Jeg forsøkte, men klarte ikke å finne et endepunkt for å søke etter journalføringer. Vet du om et? Enig i kommentaren. Jeg kan forsøke å leite litt til.

Reply:
Hadde ikke refresha siden før jeg skrev kommentaren, så jeg så ikke kommentaren din Bjørn før nå ^^